backstop/scenarios/menu.js

```diff
@@ -4,15 +4,18 @@ module.exports = [
   // Menu types
   scenario('Type Default', {
     selectors: ['#demo-default'],
+    viewports: [{ width: 800, height: 600 }],
```

Comment:
Are these newly generated images or or just changed the old ones? Because github is not showing diff button so if they are new ones then need to delete old ones (if still present).

Reply:
I think `desktop` is added when there's no viewport size provided. So you'll have to delete those old images

Reply:
Ah, ok.  I don't remember that being the case in #155.  I've removed the images that don't have `_desktop` in the file name.

Reply:
Tests are failing now.  I ran the tests again and it wants to add the non `_desktop` images back in.  I am confused what's going on here.

Reply:
Seems you removed the wrong set of images

Reply:
That fixed it.  Let's just pretend this mistake never took place. 😂